### DISCUSSION TITLE
[Modal] When onApprove called the form inside modal is destroyed

### DISCUSSION THREAD

**Discussion by djakovljevic-pr** [fomantic/Fomantic-UI#1559] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Bug Report
My modal is shown when you click on a button. The first time everything works as intended. The problems start when you click the approve button. Right before the modal is closed you can see that the form inside the modal is destroyed and "sent in the background". After that each time you click on the button to show the modal again the form is behind the modal.

## Steps to reproduce
1. Make a button
2. Make a modal with actions (Approve and Cancel)
3. Add trigger to show the modal on button click
4. Click the Approve button on the modal
5. Click again the button to show the modal

## Expected result
The form inside the modal should be inside the modal each time.

## Actual result
Just the first time the form is rendered as expected. After that the form is behind the modal.

## Testcase
https://jsfiddle.net/filozof555/8bxe53dc/5/

## Screenshot (if possible)
![first_trigger](https://user-images.githubusercontent.com/13545991/87007783-14cd3e00-c1c3-11ea-9a16-c69e39077e4a.jpg)
![second_trigger](https://user-images.githubusercontent.com/13545991/87007787-16970180-c1c3-11ea-994b-efac04b0ecd8.jpg)

## Version
jquery-3.5.1.min.js
2.8.6/dist/semantic.min.js
2.8.6/dist/semantic.min.css

**Comment by ColinFrick** [comment, marked as answer]
You are calling `$('#user_form').modal('hide');` which should be `$('#add_user_modal').modal('hide');`.

**Comment by djakovljevic-pr** [comment]
Thank you! I was looking into this for hours and didn't see that typo...

**Comment by ColinFrick** [comment]
No problem. Glad I was able to help.
